README.md

```diff
@@ -11,53 +11,112 @@ CIAOS is a general purpose KV/Object store optimised for machine learning practi
 5. Availability - Seeks contribution for design. [Discussion]()
 6. Client Library and Http Services - Client package is currently available for Python only. Both Client and API end points. - [Discussion]()
 
+---
 
 ## Getting Started
 
 ```bash
 pip install ciaos
 ```
 
-### Usage
-- API_URL: the url to the fastapi server.
-- key: the key value which can be used to identify your content in Database.
-- Value: the Base64 encoded binary data that you want to save / upload to database.
-- new_key: The new_key is an optional parameter. To be used if and only if you want to update the key.
+---
 
-### SAVE
+## Usage
 
+#### Configuration
+To use CIAOS, you need to initialize a `Config` object with the necessary details:
+
+- **`user_id`**: Your user ID .
+- **`api_url`**: The URL of the FastAPI server.
+- **`user_access_key`**: Your user access key for authentication.
+
+Here's how to set up and use the library:
+
+```python
+from ciaos import Ciaos,Config
+
+# Initialize Config
+config = Config(
+    user_id="your_user_id", 
+    api_url="https://api.ciaos.com",
+    user_access_key="xxxx"
+)
+
+# Initialize CIAOS Client
+ciaos_client = Ciaos(config)
 ```
-from ciaos import save
 
-save(API_URL, key, value)
+---
 
+### Functions and Usage
+
+#### **PUT**
+Uploads a file to the server.
```

Comment:
Either Describe everything in the begining or at the end. It will be consuming for people, We''l have text at the bottom and code at end. Follow the same pattern everywhere